IfWrapper/Common/i2c_common.h

```diff
@@ -0,0 +1,61 @@
+/**
+ * @file
+ * @brief I2C 通信関連の一般 IO 以外の公開関数
+ * @note レジスタの読み書き等，ハードウェアに依存しない関数群を定義する
+ */
+
+#ifndef I2C_COMMON_H_
+#define I2C_COMMON_H_
+
+#include "../../Drivers/Super/driver_super.h"
+#include "../i2c.h"
+
+/**
+ * @brief I2C_Config 構造体にて指定されたデバイスのレジスタへ 1 バイトデータを書き込みます
+ * @param[in] p_super  : DriverSuper 構造体へのポインタ
+ * @param[in] stream   : 使用する stream_config の番号
+ * @param[in] my_i2c_v : 対象とする I2C_Config 構造体へのポインタ
+ * @param[in] register_address : 書き込むレジスタのアドレス
+ * @param[in] data     : 書き込むデータ
+ * @retval DS_ERR_CODE
```

Comment:
```suggestion
 * @retval DS_ERR_CODE (DS_send_general_cmd の返り値)
```

とあると親切そう（そして wrap してることがわかって良さそう）